application/actions/meetings_fixtures_test.go

```diff
@@ -0,0 +1,69 @@
+package actions
+
+import (
+	"github.com/gobuffalo/nulls"
```

Comment:
I found in Goland's settings for the code editor (Code Style) you can tell it to sort and group import statements.  It looks like that will work when it automatically adds an import statement.

I doubt if CI/CD could do it, since that would require an automated commit-push to the repo, right?